slave/buildslave/commands/utils.py

```diff
@@ -19,6 +19,8 @@
 from twisted.python.procutils import which
 from twisted.python import runtime
 
+import klog
```

Comment:
Minor: It would be nice to not introduce new pep issues. My recommendation is to add an extra blank line between import statements and class declaration

Reply:
Done